tests/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Test/Verifiers/VisualBasicCodeFixVerifier.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Testing;
+using Microsoft.CodeAnalysis.Testing.Verifiers;
+using Microsoft.CodeAnalysis.VisualBasic.Testing;
+
+#pragma warning disable CA1000 // Do not declare static members on generic types
+#pragma warning disable CA1034 // Nested types should not be visible
+#pragma warning disable CA1724 // The type name Test conflicts
+
+namespace Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Test
```

Comment:
seems to be a lot of test plumbing code in this PR. is there enough to make a test.common library?

Reply:
Maybe - I'm planning on seeing if we can just integrate some of it into the Microsoft.CodeAnalysis.Testing library itself